plugins/modules/packaging/language/maven_artifact.py

```diff
@@ -150,6 +150,17 @@
         default: 'md5'
         choices: ['md5', 'sha1']
         version_added: 3.2.0
+    unredirected_headers:
+        type: list
+        default: ['Authorization', 'Cookie']
+        elements: str
+        version_added: 5.2.0
+        description:
+            A list of headers that should not be included in the redirection. This headers are sent to the fetch_url
```

Comment:
It's better to split this up into paragraphs for readability.
```suggestion
          - A list of headers that should not be included in the redirection. This headers are sent to the fetch_url
```